src/pages/DataResearch.tsx

```diff
@@ -1,11 +1,125 @@
+import React, { useMemo } from 'react'
+import { useGroupBy } from 'src/api/groupByService'
 import Widget from 'src/shared/Widget'
+import { useDate } from './components/DateTimePicker'
+import moment from 'moment'
+import { useState } from 'react'
+import { Grid } from '@mui/material'
+import { DateSelector } from './components/DateSelector'
+import { useTranslation } from 'react-i18next'
+import { Skeleton } from 'antd'
+import { Area, AreaChart, CartesianGrid, ResponsiveContainer, XAxis, YAxis } from 'recharts'
+import { Tooltip } from 'react-leaflet'
+import { PageContainer } from './components/PageContainer'
+import { getColorName } from './dashboard/AllLineschart/OperatorHbarChart/OperatorHbarChart'
+
+const now = moment()
+const unique: (value: string, index: number, self: string[]) => boolean = (value, index, self) =>
+  self.indexOf(value) === index
 
 export const DataResearch = () => {
   return (
-    <>
-      <Widget>
-        <h1>מחקרים</h1>
-      </Widget>
-    </>
+    <PageContainer>
+      <h1>מחקרים</h1>
+      <StackedResearchChart />
+    </PageContainer>
+  )
+}
+
+export const StackedResearchChart = () => {
+  const [startDate, setStartDate] = useDate(now.clone().subtract(7, 'days'))
+  const [endDate, setEndDate] = useDate(now.clone().subtract(1, 'day'))
+  const [groupByHour, setGroupByHour] = React.useState<boolean>(false)
+  const { t } = useTranslation()
+
+  const [graphData, loadingGraph] = useGroupBy({
+    dateTo: endDate,
+    dateFrom: startDate,
+    groupBy: groupByHour ? 'operator_ref,gtfs_route_hour' : 'operator_ref,gtfs_route_date',
+  })
+
+  const data = useMemo(
+    () =>
+      graphData.reduce((acc, curr) => {
+        const date = curr.gtfs_route_date ?? curr.gtfs_route_hour
+        const entry = acc.find((item) => item.name === date)
+        if (entry) {
+          entry[curr.operator_ref?.agency_id || 'Unknown'] = curr.total_actual_rides
+        } else {
+          const newEntry = {
+            name: date,
+            [curr.operator_ref?.agency_id || 'Unknown']: curr.total_actual_rides,
+          }
+          acc.push(newEntry)
+        }
+        return acc
+      }, [] as Record<string, string | number>[]),
+    [graphData],
+  )
+
+  const operators = graphData
+    .map((operator) => operator.operator_ref?.agency_id || 'Unknown')
+    .filter(unique)
+
+  return (
+    <Widget>
+      <h2>בעיות etl/gps/משהו גלובאלי אחר</h2>
+      <p>אם יש לכם רעיון מעניין למה קורים פה דברים, דברו איתנו בסלאק!</p>
+      <Grid container>
+        <Grid xs={6} item>
+          <DateSelector
+            time={startDate}
+            onChange={(data) => setStartDate(data)}
+            customLabel={t('start')}
+          />
+        </Grid>
+        <Grid xs={6} item>
+          <DateSelector
+            time={endDate}
+            onChange={(data) => setEndDate(data)}
```

Comment:
you're right, but since it's an internal endpoint, it's less important here.
However, it is a great idea to open an issue to fix that on the front page